Voycar.Api.Web/Features/Members/Post/ResetPassword/Endpoint.cs

```diff
@@ -0,0 +1,49 @@
+namespace Voycar.Api.Web.Features.Members.Post.ResetPassword;
+
+using Repository;
+
+
+public class Endpoint : Endpoint<Request>
+{
+    private readonly IUsers _repository;
+    private readonly ILogger<Get.Verify.Endpoint> _logger;
+
+
+    public Endpoint(IUsers repository, ILogger<Get.Verify.Endpoint> logger)
+    {
+        this._repository = repository;
+        this._logger = logger;
+    }
+
+
+    public override void Configure()
+    {
+        this.Post("/reset-password");
+        this.AllowAnonymous();
+    }
+
+
+    public override async Task HandleAsync(Request req, CancellationToken ct)
+    {
+        // Checks whether there is a user for the req
+        var user = await this._repository.Retrieve("passwordResetToken", req.PasswordResetToken);
```

Comment:
Rider beschwert sich hier, weil `req.PasswordResetToken == null` sein kann. Also bitte entweder ein `!` dahinter oder einen richtigen `null`-Check, falls es für uns relevant passieren könnte, dass das `null` ist.

Reply:
Hab die Paramenter in der Retrieve Methode angepasst, sodass null übergeben werden darf. Wenn null übergeben wird, dann wird einfach kein passender User gefunden.